server/app.py

```diff
@@ -9,7 +9,7 @@
 @app.route('/GET', methods=['GET'])
```

Comment:
Usually, we want to name the REST API call to be a noun, like
* `/users`
* `/chat_history`

See: https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/